Makefile

```diff
@@ -205,92 +198,88 @@ test: $(TESTGOMODULES) ## Run Go unit tests
 
 ##@ Docker
 
+# Export params required in Docker Bake
+export DOCKER_REGISTRY DOCKER_TAG VERSION BUILD_TIMESTAMP COMMIT_HASH VMCLARITY_TOOLS_CLI_DOCKER_ARG=$(VMCLARITY_TOOLS_BASE)
```

Comment:
I'd prefer using `docker buildx bake --set` argmunet to provide variables to the build instead of setting this variable for all the targets in the Makefile. Like:

```makefile
BAKE_OPTS =
BAKE_OPTS += --set DOCKER_REGISTRY=$(DOCKER_REGISTRY)
BAKE_OPTS += --set DOCKER_TAG=$(DOCKER_TAG)
...
```

```Makefile
docker: ## Build All Docker images
      	$(info Building all docker images ...)
        docker buildx bake $(BAKE_OPTS)
```